src/screens/HomeScreen.js

```diff
@@ -399,47 +295,18 @@ export const HomeScreen = ({ navigation }) => {
                   );
                 }
 
-                const eventRecords =
-                  data &&
-                  data.eventRecords &&
-                  data.eventRecords.map((eventRecord, index) => ({
-                    id: eventRecord.id,
-                    subtitle: subtitle(
-                      eventDate(eventRecord.listDate),
-                      !!eventRecord.addresses &&
-                        !!eventRecord.addresses.length &&
-                        (eventRecord.addresses[0].addition || eventRecord.addresses[0].city)
-                    ),
-                    title: eventRecord.title,
-                    picture: {
-                      url: mainImageOfMediaContents(eventRecord.mediaContents)
-                    },
-                    routeName: 'Detail',
-                    params: {
-                      title: 'Veranstaltung',
-                      query: QUERY_TYPES.EVENT_RECORD,
-                      queryVariables: { id: `${eventRecord.id}` },
-                      rootRouteName: 'EventRecords',
-                      shareContent: {
-                        message: shareMessage(eventRecord, QUERY_TYPES.EVENT_RECORD)
-                      },
-                      details: eventRecord
-                    },
-                    bottomDivider: index !== data.eventRecords.length - 1,
-                    __typename: eventRecord.__typename
-                  }));
+                const eventRecords = parseEventRecords(data?.[QUERY_TYPES.EVENT_RECORDS], true);
 
                 if (!eventRecords || !eventRecords.length) return null;
 
                 const eventRecordsListType = listTypesSettings[QUERY_TYPES.EVENT_RECORDS];
-                const ListComponent = getListComponent(eventRecordsListType);
 
                 return (
                   <View>
                     <ListComponent
                       navigation={navigation}
                       data={eventRecords}
-                      leftImage={eventRecordsListType === LIST_TYPES.IMAGE_TEXT_LIST}
```

Comment:
nice that this `leftImage` is not necessary anymore

Reply:
I think it was unnecessary before, due to the way the `ImageTextList` was either selected In the case were the condition is `true` and then ignores the prop altogether.